pkg/lib/interaction/graph.go

```diff
@@ -243,19 +245,25 @@ func (g *Graph) GetACR(amrValues []string) string {
 	return ""
 }
 
+func (g *Graph) FillDetails(err error) error {
+	return errorutil.WithDetails(err, errorutil.Details{
+		"IntentKind": apierrors.APIErrorDetail.Value(IntentKind(g.Intent)),
```

Comment:
I found some of the validation is done in handler, so `IntentKind` will be missing for the error in that case. e.g. For signup with phone number, we parse the phone number by function `FormToLoginID` in handler.